gcp_variant_transforms/beam_io/vcfio.py

```diff
@@ -715,3 +847,56 @@ def __init__(
 
   def expand(self, pvalue):
     return pvalue.pipeline | Read(self._source)
+
+
+class WriteToVcf(PTransform):
+  """A PTransform for writing to VCF files."""
+
+  def __init__(self,
+               file_path_prefix,
+               file_name_suffix='',
```

Comment:
I think the interface is cleaner if we don't provide this option. It will be .vcf unless compression_type is GZIP/BZIP (in which case we can change it).

Reply:
Should we actually pass '.vcf' into the super constructor? Or should we just assume the user passes in "filename.vcf" as the argument to `file_path`?